css/style.css

```diff
@@ -417,4 +417,139 @@ body{
         width: 25%;
     }
 }
-/***** End services *****/
\ No newline at end of file
+/***** End services *****/
+/***** client *****/
+.client{
+    padding: 3.75rem 0;
+}
+.client .flex-container{
```

Comment:
I recomend you to stick to single naming style: camelCase or using '-' symbol to separate words

Reply:
I opened this html locally and found that header looks like this when you scroll down since it's position is fixed. Is it ok from UX point of view? Possibly it makes sense to make header background more dark so it will look more clear on white background?

<img width="1673" alt="Screenshot 2022-03-07 at 19 25 17" src="https://user-images.githubusercontent.com/17166011/157085778-938fdfbc-a67c-40ea-8fae-83367104d6a3.png">